docs/resources/git_trigger.md

```diff
@@ -0,0 +1,65 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "octopusdeploy_git_trigger Resource - terraform-provider-octopusdeploy"
+subcategory: ""
+description: |-
+  This resource manages Git repository triggers in Octopus Deploy.
+---
+
+# octopusdeploy_git_trigger (Resource)
+
+This resource manages Git repository triggers in Octopus Deploy.
+
+## Example Usage
+
+```terraform
+resource "octopusdeploy_git_trigger" "my_trigger" {
+  name                = "My Git trigger"
+  space_id            = "Spaces-1"
+  project_id          = "Projects-1"
+  channel_id          = "Channels-1"
+  sources {
+    deployment_action_slug = "deploy-action-slug"
+		git_dependency_name    = ""
+		include_file_paths     = ["include/me", "include/me/too"]
+		exclude_file_paths     = ["exclude/me", "exclude/me/too"]
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `channel_id` (String) The ID of the channel in which the release will be created if the action type is CreateRelease.
+- `name` (String) The name of this resource.
+- `project_id` (String) The ID of the project to attach the trigger.
+
+### Optional
+
+- `is_disabled` (Boolean) Disables the trigger from being run when set.
+- `sources` (Block List) List of Git trigger sources. Contains details of the deployment action slug, the git dependency and what file paths to monitor. (see [below for nested schema](#nestedblock--sources))
```

Comment:
Are Git trigger sources optional? What happens if you supply an empty list, do we allow that? I suppose it just doesn't do anything which might be ok.